apps/api/test/report-util.ts

```diff
@@ -103,13 +104,13 @@ export const getIndexOfStatus = (arr: unknown[], compareTo: string) => {
   return arr.findIndex((v: any) => v.status === compareTo);
 };
 
-export const getStatusId = async (status: STATUS): Promise<string> => {
+export const getStatusId = async (status: STATUS | ReportTestStatus): Promise<string> => {
```

Comment:
shouldn't STATUS be a superset of ReportTestStatus?

Reply:
I see what you mean here. Resolving this currently.